From a9265e86ddbaa96f8ea1ee03f66197ab2e5a6674 Mon Sep 17 00:00:00 2001
From: Jianzheng Zhou <jianzheng.zhou@freescale.com>
Date: Sun, 6 Jan 2013 09:52:35 +0800
Subject: [PATCH 034/231] ENGR00180079 rfkill:revert "change rfkill node
 permission"

This reverts commit 60daab290bbab00c73cc057ff868f658ec73d304.
For it will break android CTS. Change to fix it in upper.

Signed-off-by: Jianzheng Zhou <jianzheng.zhou@freescale.com>
---
 net/rfkill/core.c |    9 ---------
 1 file changed, 9 deletions(-)

diff --git a/net/rfkill/core.c b/net/rfkill/core.c
index 9a2d2a0..df2dae6 100644
--- a/net/rfkill/core.c
+++ b/net/rfkill/core.c
@@ -2,7 +2,6 @@
  * Copyright (C) 2006 - 2007 Ivo van Doorn
  * Copyright (C) 2007 Dmitry Torokhov
  * Copyright 2009 Johannes Berg <johannes@sipsolutions.net>
- * Copyright (C) 2012 Freescale Semiconductor, Inc.
  *
  * This program is free software; you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
@@ -720,18 +719,10 @@ static ssize_t rfkill_claim_store(struct device *dev,
 
 static struct device_attribute rfkill_dev_attrs[] = {
 	__ATTR(name, S_IRUGO, rfkill_name_show, NULL),
-#ifdef CONFIG_ANDROID_PARANOID_NETWORK
-	__ATTR(type, S_IRUGO|S_IWUGO, rfkill_type_show, NULL),
-#else
 	__ATTR(type, S_IRUGO, rfkill_type_show, NULL),
-#endif
 	__ATTR(index, S_IRUGO, rfkill_idx_show, NULL),
 	__ATTR(persistent, S_IRUGO, rfkill_persistent_show, NULL),
-#ifdef CONFIG_ANDROID_PARANOID_NETWORK
-	__ATTR(state, S_IRUGO|S_IWUGO, rfkill_state_show, rfkill_state_store),
-#else
 	__ATTR(state, S_IRUGO|S_IWUSR, rfkill_state_show, rfkill_state_store),
-#endif
 	__ATTR(claim, S_IRUGO|S_IWUSR, rfkill_claim_show, rfkill_claim_store),
 	__ATTR(soft, S_IRUGO|S_IWUSR, rfkill_soft_show, rfkill_soft_store),
 	__ATTR(hard, S_IRUGO, rfkill_hard_show, NULL),
-- 
1.7.10.4

